testing/regress/ecl/aggidx1.ecl

```diff
@@ -24,6 +24,8 @@ useSequential := #IFDEFINED(root.useSequential, false);
 
 //--- end of version configuration ---
 
+#option ('obfuscateOutput', true);
```

Comment:
In this case I wanted to use a test which used indexes since that would be something that might be broken by removing meta information from the graph.  If I had added a new test it would have essentially been a clone of one of the existing ones - so wouldn't have had any benefit.

If it is testing a new piece of functionality then I agree.  This isn't though.